packages/sdk-v2/src/sdk/analytics/amplitude/api.native.ts

```diff
@@ -28,11 +28,11 @@ class Wrapper {
     this.api.setUserId(id);
   }
 
-  setUserProperties(props: object) {
+  setUserProperties(props: any) {
```

Comment:
why 'any' ? if object doesn't works, could we use something like key-value dictionary (e.g. `Record<string, any>`) ?